### PR TITLE
Makes the CentCom Uniform available in a hacked capdrobe

### PR DESCRIPTION
**Pull request by Sir-Lagsalot**
Someone wanted it, seemed like it might be fun.

:cl:  
tweak: You can now buy a CentCom uniform from the captain's clothes vendor if you hack it.
/:cl:

### CODE REVIEW

File: code/modules/vending/autodrobe.dm [hunk 1]
```diff
@@ -256,6 +256,7 @@
 					/obj/item/clothing/head/yogs/whitecaptaincap = 1,
 					/obj/item/clothing/under/yogs/victoriouscaptainuniform = 1,
 					/obj/item/clothing/head/beret/captain = 1)
+	contraband = list(/obj/item/clothing/under/rank/centcom_officer = 2)
```
Comment (alexkar598) [inline code/modules/vending/autodrobe.dm:259]:
```suggestion
	contraband = list(/obj/item/clothing/under/rank/centcom_officer = 1)
```

**Comment by MegaEmpirical** [comment]
Personally not a fan of it. I prefer it to be really rare, that's how the gimmick works. Plus can't you just use the chameleon kit?

**Comment by swissloaf** [comment]
this ruins the point of buying it with telecrystals

**Comment by LoliconSlayer** [comment]
Seems like it's asking for dumbfuck captains to set someone's ID to "CentComm Official" and give them the uniform.

**Comment by MegaEmpirical** [comment]
... Fuck, they would too.

**Comment by alexkar598** [comment]
What the tomato said

**Comment by Sir-Lagsalot** [comment]
Yeah fair enough

**Pull request closed without merging**